src/components/environment-variables/environment-variables-list.js

```diff
@@ -0,0 +1,287 @@
+                                  title={getOriginalEnvVarToolTip(envVar)}
+                                >
+                                  <Icon data={layers} />
+                                </Tooltip>
+                                {envVar.metadata.radixConfigValue &&
+                                  envVar.metadata.radixConfigValue.length >
+                                    0 && (
+                                    <Tooltip
+                                      enterDelay={0}
+                                      placement="right"
+                                      title={getOriginalEnvVarToolTip(envVar)}
+                                    >
+                                      <Label
+                                        label={envVar.metadata.radixConfigValue}
+                                      />
+                                    </Tooltip>
+                                  )}
+                              </div>
+                            )}
+                            <div className="form-field">
+                              <Input
+                                id={'envVar' + envVar.name}
+                                disabled={
+                                  !inEditMode ||
+                                  saveState.status === requestStates.IN_PROGRESS
+                                }
+                                type="text"
+                                value={editableEnvVar.currentValue}
+                                onChange={(ev) =>
+                                  setEditableEnvVars(() => {
+                                    editableEnvVars[index].currentValue =
+                                      ev.target.value;
+                                    return [...editableEnvVars];
+                                  })
+                                }
+                              />
+                            </div>
+                          </Table.Cell>
+                        </Table.Row>
+                      );
+                    }
+                    if (includeRadixVars === true) {
+                      return (
+                        <Table.Row key={envVar.name}>
+                          <Table.Cell>* {envVar.name}</Table.Cell>
+                          <Table.Cell>
+                            <strong>{envVar.value}</strong>
+                          </Table.Cell>
+                        </Table.Row>
+                      );
+                    }
+                    return '';
+                  })}
+              </Table.Body>
+            </Table>
+            {saveState.status === requestStates.IN_PROGRESS && (
+              <>
+                <CircularProgress size="24" />
+                <span className="progress">Updating…</span>
+              </>
+            )}
+          </div>
+        </form>
+      )}
+    </React.Fragment>
+  );
+};
+
+export default EnvironmentVariablesList;
```

Comment:
Would be useful to define `.propTypes` for the parameters of EnvironmentVariablesList

Reply:
Added definitions of params